src/ucp/core/ucp_mm.c

```diff
@@ -106,6 +106,26 @@ static ucs_status_t ucp_memh_reg_mds(ucp_context_h context, ucp_mem_h memh,
     return UCS_OK;
 }
 
+ucs_status_t ucp_memory_type_detect_mds(ucp_context_h context, void *addr, size_t length,
+                                        uct_memory_type_t *mem_type)
+{
+    unsigned md_index;
```

Comment:
s/md_index/i
because "md_index" is usually the index in tl_mds array

Reply:
Fixed